### PR TITLE
Added v_inner and v_outer, enable_nonhomologous_expansion for future use in nonhomologous expansion

### PR DESCRIPTION
**Pull request by sonachitchyan**
**Description**
This PR adds v_inner and v_outer to NumbaModel as arguments, taking them from MonteCarloRunner. This will be used in the future for the implementation of the nonhomologous expansion.

**Motivation and context**

**How has this been tested?**

- [x] Testing pipeline
- [ ] Other

**Examples**

**Type of change**

- [ ] Bug fix 
- [x] New feature 
- [ ] Breaking change 
- [ ] None of the above 

**Checklist**

- [ ] I have updated the documentation according to my changes
- [ ] I have built the documentation by applying the `build_docs` label to this pull request (if you don't have enough privileges a reviewer will do it for you)
- [ ] I have requested two reviewers for this pull request

### CODE REVIEW

**Comment by Rodot-** [comment]
Please make sure your changes are PEP 8 compliant, you can just run `black` on the files you modified and it will do it for you

**Review by Rodot- — APPROVED** [review]

File: tardis/montecarlo/montecarlo_numba/r_packet_transport.py [hunk 1]
```diff
@@ -57,7 +57,6 @@ def trace_packet(r_packet, numba_model, numba_plasma, estimators):
     tau_trace_line_combined = 0.0
 
     # e scattering initialization
-
```
Comment (andrewfullard) [inline tardis/montecarlo/montecarlo_numba/r_packet_transport.py:60]: Shouldn't be part of the PR

File: tardis/montecarlo/montecarlo_numba/tests/test_packet.py [hunk 2]
```diff
@@ -31,6 +29,8 @@ def model():
     return numba_interface.NumbaModel(
         r_inner=np.array([6.912e14, 8.64e14], dtype=np.float64),
         r_outer=np.array([8.64e14, 1.0368e15], dtype=np.float64),
+        v_inner=np.array([6.912e14, 8.64e14], dtype=np.float64),
+        v_outer=np.array([8.64e14, 1.0368e15], dtype=np.float64),
```
Comment (andrewfullard) [inline tardis/montecarlo/montecarlo_numba/tests/test_packet.py:33]: Since these match the radius values, I assume they do not affect the tests. In which case it would be good to put nonsense values in like -1 so it's clear they are not significant.

**Review by andrewfullard — CHANGES_REQUESTED** [review]

File: tardis/montecarlo/montecarlo_numba/tests/test_cuda_formal_integral.py [hunk 3]
```diff
@@ -117,7 +117,7 @@ def formal_integral_model(request):
     This gets the Numba model to be used in later tests
     """
     r = request.param["r"]
-    model = NumbaModel(r[:-1], r[1:], 1 / c.c.cgs.value)
+    model = NumbaModel(r[:-1], r[1:], r[:-1], r[1:], 1 / c.c.cgs.value)
```
Comment (wkerzendorf) [inline tardis/montecarlo/montecarlo_numba/tests/test_cuda_formal_integral.py:120]: please change to velocities

**Review by wkerzendorf — COMMENTED** [review]
Please change as discussed

**Review by andrewfullard — APPROVED** [review]
